Makefile

```diff
@@ -0,0 +1,33 @@
+GO           ?= go
+GOFMT        ?= $(GO)fmt
+FIRST_GOPATH := $(firstword $(subst :, ,$(shell $(GO) env GOPATH)))
```

Comment:
`FIRST_GOPATH` seems unused unless I'm missing something?

Reply:
You're right, this was used for something that I ended up not doing.